### PR TITLE
Update Ruby version

### PR DESCRIPTION
**Pull request by srisankethu**
Fixes https://github.com/dfarrell07/puppet-opendaylight/issues/119

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +26,14 @@ matrix:
...
   - rvm: 2.2.3
     env: PUPPET_VERSION="~> 3.7.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.4.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.5.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.6.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.7.0"
```
Comment (dfarrell07) [inline .travis.yml]: I think we should remove these old versions of Puppet. https://github.com/dfarrell07/puppet-opendaylight/commit/58fd3bfdd4be4f3c24679d57ca7bb7a4e4f2d833#diff-354f30a63fb0907d4ad57269548329e3R18

File: .travis.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@ language: ruby
 bundler_args: --without local_only
 before_install: rm Gemfile.lock || true
 rvm:
-  - 2.0.0
   - 2.1.7
   - 2.2.3
```
Comment (dfarrell07) [inline .travis.yml]: I think there are newer 2.2.* releases

**Comment by dfarrell07** [comment]
Ah, I see this is just a PR for Ruby versions. Maybe you want to do #119 in two commits, one for Ruby and one for Puppet? That's fine with me I guess. Let's do the 2.2.6 upgrade here tough https://www.ruby-lang.org/en/downloads/

**Comment by srisankethu** [comment]
@dfarrell07 I have added newer 2.2.6 version and remove the exceptions

**Comment by srisankethu** [comment]
@dfarrell07 Errors Alert! I will fix it in a separate patch. Anyway, I am trying to improve the test-coverage.

**Comment by dfarrell07** [comment]
@srisankethu ACK, thanks :+1:

**Comment by srisankethu** [comment]
@dfarrell07 This is strange! The tests are passing locally when I am executing `bundle exec rake test` for `Ruby: 2.3.1` and `Puppet: 4.8.1` but failing in the build :(

File: .travis.yml [hunk 3]
```diff
@@ -3,29 +3,16 @@ language: ruby
...
-  exclude:
-  - rvm: 2.2.3
-    env: PUPPET_VERSION="~> 3.4.0"
-  - rvm: 2.2.3
-    env: PUPPET_VERSION="~> 3.5.0"
-  - rvm: 2.2.3
-    env: PUPPET_VERSION="~> 3.6.0"
-  - rvm: 2.2.3
-    env: PUPPET_VERSION="~> 3.7.0"
+  - PUPPET_VERSION="~> 4.5.0"
```
Comment (dfarrell07) [inline .travis.yml]: We should do the latest minor/patch versions from the 3.* major version as well

**Comment by dfarrell07** [comment]
:+1:

**Pull request merged**